Makefile

```diff
@@ -53,6 +54,7 @@ style:
 # run tests for the repo
 test:
 	@echo "Running python tests";
+	export SPARSEZOO_TEST_MODE=$(SPARSEZOO_TEST_MODE);
```

Comment:
do we need to unset this after testing?

Reply:
I actually set the variable at the top of the makefile so it is only within the context of the makefile. This line isn't necessary but I forgot to delete it.

Reply:
oh nice, I didn't realize the Makefile vars would be passed to the environment.  that's neat.